sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/DataSourceStrategy.scala

```diff
@@ -321,8 +321,8 @@ private[sql] object DataSourceStrategy extends Strategy with Logging {
     val metadata: Map[String, String] = {
       val pairs = ArrayBuffer.empty[(String, String)]
 
-      if (pushedFilters.nonEmpty) {
-        pairs += (PUSHED_FILTERS -> pushedFilters.mkString("[", ", ", "]"))
+      if (handledPredicates.nonEmpty) {
+        pairs += (HANDLED_FILTERS -> handledPredicates.mkString("[", ", ", "]"))
```

Comment:
ah sorry. I think I understand the change now. `handledPredicates` contains all filters that are pushed to the data source except those filters returned by the unhandledFilters method. I think the change is good and `HandledFilters` is a proper name.